backend/migrations/000002_checkpoints_update_schema.up.sql

```diff
@@ -1,11 +1,41 @@
-ALTER TABLE checkpoints
-    ADD COLUMN IF NOT EXISTS thread_ts TIMESTAMPTZ,
-    ADD COLUMN IF NOT EXISTS parent_ts TIMESTAMPTZ;
+-- Drop existing checkpoints-related tables if they exist
+ALTER TABLE IF EXISTS checkpoints RENAME TO old_checkpoints;
+DROP TABLE IF EXISTS checkpoint_writes;
+DROP TABLE IF EXISTS checkpoint_blobs;
```

Comment:
is dropping any of these checkpoints resulting in dataloss? If so should we back them up as well?